### PR TITLE
Allow specifiying an FAQ link when dissolving parliament

### PR DESCRIPTION
**Pull request by h-lame**
If present this link will be displayed at the bottom of the message as follows:

> Find out more on the **Petitions Committee website**

Where the part in bold will be the clickable link.  If no url is provided that sentence is not included in the warning notification at all.

### CODE REVIEW

File: app/views/application/_parliament_dissolution_warning.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<% if Parliament.dissolution_announced? %>
+  <div class="notification">
+    <span class="icon icon-warning-white"></span>
+    <h3 class="header"><%= Parliament.dissolution_heading %></h3>
+    <p class="content"><%= Parliament.dissolution_message %></p>
+    <% if Parliament.dissolution_faq_url.present? %>
```
Comment (pixeltrix) [inline app/views/application/_parliament_dissolution_warning.html.erb]: The `.present?` is unnecessary - the query attribute method is enough

**Comment by pixeltrix** [comment]
:shipit:  the present change isn't worth bothering with

File: app/views/application/_parliament_dissolution_warning.html.erb (hunk 1 above)
Comment (h-lame) [inline app/views/application/_parliament_dissolution_warning.html.erb]: I always forget about those. I have a mental block on them.

**Comment by h-lame** [comment]
Made the query attribute change, also added `:site_updated_at` to the fragments used to invalidate the cache on the homepage.  Without this changing the Parliament model wouldn't refresh the home page block where the "parliament dissolved" notification now lives.

**Pull request merged**